ports/tts/portfile.cmake

```
@@ -0,0 +1,25 @@
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO jfalcou/tts
+    REF 2.2
+    SHA512 d3c17a4490f483bee724578301be17f042df7aa996c9309347875b54b25c868f5af6dcd72c15839f48f16973a564d578590f1ae517045053fbdb5b244860db67
+    HEAD_REF main
+)
+
+set(VCPKG_BUILD_TYPE release) # header-only port
+
+vcpkg_cmake_configure( SOURCE_PATH "${SOURCE_PATH}"
+                       OPTIONS -DTTS_BUILD_TEST=OFF
+                     )
```

```suggestion
vcpkg_cmake_configure(
    SOURCE_PATH "${SOURCE_PATH}"
    OPTIONS
        -DTTS_BUILD_TEST=OFF
        -DCMAKE_DISABLE_FIND_PACKAGE_Python=ON
)
```

Please also disable using Python because the resulting build should not depend on weither certain dependencies are installed.

---

Python is only used for CI and internal development. NOthing uses it in the install process.
If youdont have Python installed the CI related targets are disabled, that's all.

---

But let's disable it.